packages/desktop/src/index.html

```diff
@@ -1,22 +1,15 @@
 <!DOCTYPE html>
 <html>
-
-<html>
-
-<head>
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" charset="utf-8" />
-</head>
-
-
-<body>
-    <div id="main">
-    </div>
-</body>
-
-<script>
-    import("./index.js").then(function (module) {
-        module.main();
-    });
-</script>
-
+    <head>
+        <title>Dioxus app</title>
```

Comment:
This is not strictly necessary for dioxus, but turns this file into a fully valid document.
It get's overwritten by dioxus or the user's application configuration anyway.

Reply:
I don't know should we add this `title` tag. but maybe it doesn't matter.